packages/wdio-browserstack-service/src/service.ts

```diff
@@ -179,6 +221,18 @@ export default class BrowserstackService implements Services.ServiceInstance {
 
             this._failReasons.push(exception)
         }
+
+        await this._insightsHandler?.afterScenario(world)
+    }
+
+    async beforeStep (step: Frameworks.PickleStep, scenario: Pickle) {
+        const { keyword, text } = step
+        await this._insightsHandler?.beforeStep(step, scenario)
+        await this._setAnnotation(`Step: ${keyword}${text}`)
```

Comment:
There is not really any value pulling out properties out of an object if these are only used once. It only blows up the code base.
```suggestion
        await this._insightsHandler?.beforeStep(step, scenario)
        await this._setAnnotation(`Step: ${step.keyword}${step.text}`)
```